src/main/java/org/folio/ld/fingerprint/config/FingerprintEntrySerializer.java

```diff
@@ -0,0 +1,27 @@
+package org.folio.ld.fingerprint.config;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+import java.io.IOException;
+import org.folio.ld.fingerprint.service.FingerprintServiceImpl.FingerprintEntry;
+
+
+public class FingerprintEntrySerializer extends StdSerializer<FingerprintEntry> {
+
+  public FingerprintEntrySerializer() {
+    this(null);
```

Comment:
Hi @askhat-abishev, 

Should we pass `FingerprintEntry.class` instead of `null`?

Or, is it enough to have a single constructor as below?
```
 FingerprintEntrySerializer() {
    super(FingerprintEntry.class);
}
```

Reply:
Hi @pkjacob either option will work the same